index.html

```diff
@@ -0,0 +1,33 @@
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>simpleweb</title>
+    </head>
+    <body>
+        <header>
+        <div class="toolbar">
+            <div class="logo">My Logo</div>
+            <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+        </div>
+        </header>
+    <section>
+         <div> 
+        <h1 class="header">I'm Mariane<br>Glad to see you!</h1></div>    
+      <div>  <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    </section>
+        <footer class="contact">
+            <p class="connect">LET'S CONNECT</p>
```

Comment:
This is a link to the `Contact` section of the page, so don't you think it's not a paragraph?